### PR TITLE
Proper handling of ssl certificate validation

### PR DESCRIPTION
**Pull request by tardieu**
Only ignore ssl certificate validation failures when explicitly configured to do so.

### CODE REVIEW

File: conductor.js [hunk 1]
```diff
@@ -60,8 +61,9 @@ module.exports = function (options) {
 
   if (process.env.__OW_API_HOST) apihost = process.env.__OW_API_HOST
   if (process.env.__OW_API_KEY) apikey = process.env.__OW_API_KEY
+  if (process.env.__OW_IGNORE_CERTS) ignorecerts = process.env.__OW_IGNORE_CERTS
```
Comment (rabbah) [inline conductor.js]: this variable is not actually set by the invoker (like the other two) if you're relying on that.

Comment (tardieu) [reply conductor.js]: It is only used for deployment. For running compositions, I use an invocation parameter `{ $composer: { openwhisk: { ignore_certs: true } } }` as documented in the updated readme.

**Comment by tardieu** [comment]
I forgot to update the travis configuration. Force pushed an update.

**Comment by tardieu** [comment]
Fixes #13.

**Pull request merged**